shared/javascripts/local_storage.js

```diff
@@ -24,7 +24,7 @@ var ls = {
     }
 
     if ( ls.localStorageDefined ) {
-      return localStorage[key] = value;
+      return localStorage.setItem(key, value);
```

Comment:
That is a really good catch.  I'll make the change so that the behavior stays the same.